openmp/libomptarget/test/offloading/fortran/basic-target-parallel-region.f90

```diff
@@ -0,0 +1,21 @@
+! Basic offloading test with a target region
+! REQUIRES: flang
+! UNSUPPORTED: nvptx64-nvidia-cuda-LTO
+! UNSUPPORTED: aarch64-unknown-linux-gnu
+! UNSUPPORTED: aarch64-unknown-linux-gnu-LTO
+! UNSUPPORTED: x86_64-pc-linux-gnu
+! UNSUPPORTED: x86_64-pc-linux-gnu-LTO
```

Comment:
nit: can we replace this with:
```
! REQUIRES: flang, amdgcn-amd-amdhsa
```
?

Reply:
If we specify requirements as follows: `REQUIRES: flang, amdgcn-amd-amdhsa` then we will launch this test only for flang and AMDGPU (source: https://llvm.org/docs/TestingGuide.html ) . If we skip `amdgcn-amd-amdhsa` requirement, then we can launch the test with Flang + ` nvptx64-nvidia-cuda`.

Reply:
> If we specify requirements as follows: `REQUIRES: flang, amdgcn-amd-amdhsa` then we will launch this test only for flang and AMDGPU (source: https://llvm.org/docs/TestingGuide.html ) . If we skip `amdgcn-amd-amdhsa` requirement, then we can launch the test with Flang + ` nvptx64-nvidia-cuda`.

Ah, my bad! I didn't notice the `LTO` part in `nvptx64-nvidia-cuda-LTO`!